sphinx_rtd_theme/layout.html

```diff
@@ -20,8 +20,10 @@
   {%- endblock -%}
 
   {#- CSS #}
-  <link rel="stylesheet" href="{{ pathto('_static/' + style, 1) }}" type="text/css" />
-  <link rel="stylesheet" href="{{ pathto('_static/pygments.css', 1) }}" type="text/css" />
+  {%- if (sphinx_version <= "4.0.0") -%}
```

Comment:
Is this supposed to be `< "4.0.0"`?

(Although, even more strictly speaking, this should be a version comparison rather than a string comparison.)